server/notification-providers/hostedsms.js

```diff
@@ -1,53 +1,53 @@
-const NotificationProvider = require("./notification-provider");
-const axios = require("axios");
-const { getMonitorRelativeURL, UP, DOWN } = require("../../src/util");
-
-class HostedSMS extends NotificationProvider {
-
-    name = "hostedsms";
-
-    async send(notification, msg, monitorJSON = null, heartbeatJSON = null) {
-        let okMsg = "Sent Successfully.";
-
-        try {
-            let config = {
-                headers: {
-                    "Content-Type": "application/json",
-                }
-            };
-
-            let textMsg = "";
-            if (heartbeatJSON && heartbeatJSON.status === UP) {
-                textMsg = `✅ [${monitorJSON.name}] is back online`;
-            } else if (heartbeatJSON && heartbeatJSON.status === DOWN) {
-                textMsg = `🔴 [${monitorJSON.name}] went down`;
-            }
-
-            let data = {
-                "UserEmail": notification.hostedsmsUsername,
-                "Password": notification.hostedsmsPassword,
-                "Phone": notification.hostedsmsPhoneNumber,
-                "Message": textMsg,
-                "Sender": notification.hostedsmsSenderName,
-            };
-
-            let resp = await axios.post("https://api.hostedsms.pl/SimpleApi", data, config);
-
-            if (!resp.data.MessageId) {
-                if (resp.data.ErrorMessage) {
-                    let error = `HostedSMS.pl API returned error message: ${resp.data.ErrorMessage}`;
-                    this.throwGeneralAxiosError(error);
-                } else {
-                    let error = "HostedSMS.pl API returned an unexpected response";
-                    this.throwGeneralAxiosError(error);
-                }
-            }
-
-            return okMsg;
-        } catch (error) {
-            this.throwGeneralAxiosError(error);
-        }
-    }
-}
-
-module.exports = HostedSMS;
+const NotificationProvider = require("./notification-provider");
+const axios = require("axios");
+const { getMonitorRelativeURL, UP, DOWN } = require("../../src/util");
+
+class HostedSMS extends NotificationProvider {
+
+    name = "hostedsms";
+
+    async send(notification, msg, monitorJSON = null, heartbeatJSON = null) {
+        let okMsg = "Sent Successfully.";
+
+        try {
+            let config = {
+                headers: {
+                    "Content-Type": "application/json",
+                }
+            };
+
+            let textMsg = "";
+            if (heartbeatJSON && heartbeatJSON.status === UP) {
+                textMsg = `✅ [${monitorJSON.name}] is back online`;
+            } else if (heartbeatJSON && heartbeatJSON.status === DOWN) {
+                textMsg = `🔴 [${monitorJSON.name}] went down`;
+            }
```

Comment:
I did it like here: https://github.com/louislam/uptime-kuma/blob/master/server/notification-providers/alertnow.js
Idea is to prevent the user from paying additional costs for subsequent SMS messages unnecessarily.
Alert from ping output takes over 360 characters. In addition, with the unicode character (emoji) number of characters in a single SMS is getting smaller. In case of my test messages, a single PING down alert was counted as 6 separate SMS.

Uptime-kuma should implement shorter notification type.
Example:
```
[Example service] [🔴 Down] PING 10.1.1.111 Destination Host Unreachable
```
instead this:
```
[Example service] [🔴 Down] PING 10.1.1.111 (10.1.1.111) 56(84) bytes of data.
From 10.1.1.123 icmp_seq=1 Destination Host Unreachable
From 10.1.1.123 icmp_seq=2 Destination Host Unreachable
From 10.1.1.123 icmp_seq=3 Destination Host Unreachable

--- 10.1.1.111 ping statistics ---
3 packets transmitted, 0 received, +3 errors, 100% packet loss, time 55ms
pipe 3
```

I removed `else if`.